docs/getting-started.md

```diff
@@ -21,6 +21,26 @@ The Client SDK is a set of user-facing programs to interact with Ledger and Audi
 ScalarDL (Ledger and Auditor) abstracts data as a set of assets, where each asset is composed of the history of a record identified by a key called `asset_id` and a historical version number called `age`.
 In this document, you will create a very simple application to manage an asset's status using ScalarDL Client SDK.
 
+## Install a JDK
+
+Because ScalarDL is written in Java, using Java is one of the easist ways to build your ScalarDL applications.
+In such a case, you must have one of the following Java Development Kits (JDKs) installed in your environment:
+
+- [Oracle JDK](https://www.oracle.com/java/technologies/downloads/) LTS version (8, 11, or 17)
+- [OpenJDK](https://openjdk.org/install/) LTS version (8, 11, or 17)
+
+Also, since ScalarDL is built with JDK 8, you must use JDK 8 to build your contracts.
+If you use JDK versions other than 8, specify `-release 8` option for `javac` or define what toolchain to use in your Gradle and Maven configurations.
```

Comment:
@komamitsu @choplin 
I'm not sure if this is a proper expression. PTAL!

Reply:
Revision proposal:
```
Also, since ScalarDL is built with JDK 8, you must build contracts as a JDK 8-compatible binary.
If you use JDK versions other than 8, you must configure your build tool to build the JDK 8-compatible binary.
There are several ways to specify binary compatibility, including using the `-release 8` option for `javac` or setting Gradle and Maven configurations to use the JDK 8 toolchain.
For example, to specify toolchains in your Gradle and Maven configurations, check the Gradle doc or the Maven doc. 
```

Reply:
I guess it would also be helpful to provide examples of configuration to set toolchains. It would be only a few lines, like [this](https://github.com/scalar-labs/scalardb-analytics-spark/blob/f1accdf0119b2c158c1345ef79a99a88e1dd7661/lib/build.gradle.kts#L26-L30)

Reply:
@choplin 
Thanks!
Updated the PR based on your feedback. PTAL again!
[33acaf5](https://github.com/scalar-labs/scalardl/pull/54/commits/33acaf58129f284d62598262e8ac9095c7490e27)

Reply:
Looks good! Thank you!